diff --git a/target/product/core_minimal.mk b/target/product/core_minimal.mk
index b8e04e0cac..273c938c63 100644
--- a/target/product/core_minimal.mk
+++ b/target/product/core_minimal.mk
@@ -113,9 +113,6 @@ PRODUCT_BOOT_JARS := \
     $(TARGET_CORE_JARS) \
     ext \
     framework \
-    telephony-common \
-    voip-common \
-    ims-common \
     android.hidl.base-V1.0-java \
     android.hidl.manager-V1.0-java

